v1.1.1/js/Node.js

```diff
@@ -280,12 +286,13 @@ function Node(model, config){
 	
 		var roundedValue = Math.round(self.value * 100) / 100; // Temp variable solely to cleanly display value
 		
-		// resize value text
-		self.fillSelfSizingText(ctx, r, roundedValue, 40); // Display value slightly below label.
+		if(self.displayDebug) {
+			// resize value text
+			self.fillSelfSizingText(ctx, r, roundedValue, 40); // Display value slightly below label.
```

Comment:
Do we want the value to be lumped in with the debug text? I thought that the value display was something that we genuinely wanted for the users to be able to see.

Reply:
We can maybe separate it out later on, but Erin mentioned that once the value was displayed, she couldn't help but like obsess over what the value was, and that's not the real focus of loopy. We may or may not make a separate toggle for it, but either way I think it sounds like we won't be having it on by default anymore.